package.json

```diff
@@ -2,18 +2,20 @@
   "name": "yscope-log-viewer",
   "version": "0.1.0",
   "description": "A tool tailored for viewing logs, including log formats designed by YScope.",
-  "main": "src/index.tsx",
+  "main": "src/main.tsx",
   "scripts": {
-    "analyze:size": "webpack --config webpack.prod.js --profile --json=bundle-stats.json && webpack-bundle-analyzer bundle-stats.json dist/",
-    "build": "webpack --config webpack.prod.js",
+    "analyze:size": "npx vite-bundle-visualizer",
```

Comment:
This was added to reach dev feature-parity. Also a doc was added at `optimization-guide.md` to describe the optimization strategies for bundle size and load time.